internal/local/structure_v2_test.go

```diff
@@ -44,15 +44,27 @@ func TestParseDataSources(t *testing.T) {
 
 	testRoot := filepath.Join(wd, "testdata/data_sources")
 
+	mdbAtlasStr := "mongodb-atlas"
```

Comment:
[style/meganit] :meh: ... doubtful this really buys all that much at the expense of one extra hop in `git blame`/`git history`

Reply:
Fair enough 😅